src/smif/convert/area.py

```diff
@@ -108,10 +99,26 @@ def centroids_as_features(self):
             for region in self._regions
         ]
 
-    def intersection(self, bounds):
+    def intersection(self, to_entry):
         """Return the subset of regions intersecting with a bounding box
```

Comment:
Suggest updating this comment, something like "Return the set of regions intersecting with the bounding box of `to_entry`"